qa-tests-backend/src/main/groovy/util/Helpers.groovy

```diff
@@ -139,12 +139,12 @@ class Helpers {
             Path imageScans = Paths.get(Constants.FAILURE_DEBUG_DIR).resolve("image-scans")
             new File(imageScans.toAbsolutePath().toString()).mkdirs()
 
-            Process proc = "./scripts/ci/roxctl.sh image scan -i ${image}".execute(null, new File(".."))
+            Process proc = "./scripts/ci/roxctl.sh image scan -i ${image} -a -f".execute(null, new File(".."))
```

Comment:
`-f` forces scanner to rescan whereas without it you'll get whatever was the last scan result. Assuming the image exists in central db, otherwise it'll just get a new scan.

So wouldn't getting what was in central last be more helpful for debugging?

Reply:
Yeah, that's right. Since alerts would depend on what was present in the central, the debug info should also print the same. Removed -f option